c2corg_ui/static/js/suggestiondirective.js

```diff
@@ -0,0 +1,87 @@
+goog.provide('app.suggestionDirective');
+
+/**
+ * @param {angular.$compile} $compile Angular compile service.
+ * @param {angular.$sce} $sce Angular Strict Contextual Escaping service.
+ * @return {angular.Directive} Directive Definition Object.
+ * @ngInject
+ */
+app.suggestionDirective = function($compile, $sce) {
```

Comment:
Yep, I will have a look on that. I've had a problem with partials - they didn't compile into templatecache until they were referenced in a directive with templateUrl (really!)  but now it seems to compile. I will check it and update if it works.